### PR TITLE
#162685640 User can bookmark an article

### PR DESCRIPTION
**Pull request by fahadmak**
#### What does this PR do?
This PR enables a user to bookmark an article on Author's Haven application.

#### Description of Task to be completed?
Users should be able to bookmark articles for reading later via a bookmark tab.

#### How should this be manually tested?
`git clone https://github.com/andela/Ah-frontend-guardians.git`

`cd Ah-frontend-guardians`

`git checkout ft-bookmark-article-162685640`

`npm install` to install all the projects dependencies

`npm start` to run application.

`Go to your browser and go to this link https://ah-frontend-guardians-pr-24.herokuapp.com/`
`Click on the link of the article you are interested in.`
`On the article page click on bookmark tab` 

#### What are the relevant pivotal tracker stories?
[#162685640](https://www.pivotaltracker.com/story/show/162685640)

### CODE REVIEW

**Review by myrdstom — APPROVED** [review]
LGTM

File: src/components/Bookmark/Bookmark.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        }
+
+        if (nextProps) {
+            this.setState({ bookmarked: nextProps.bookmark && nextProps.bookmark.bookmarked });
+        }
+    }
+
+    handleBookmark = (event) => {
+        event.preventDefault();
+        console.log(this.props.slug)
```
Comment (kisakyegordon) [inline src/components/Bookmark/Bookmark.js]: Remove this console.log

**Comment by Opio-Emmanuel-Omona** [comment]
@fahadmak , when a user isn't logged in and he/she clicks the bookmark icon, nothing happens.
So I think it would be better if I get a toast to login or if I'm redirected to the login page.

Also kindly provide the link to the review app in your PR description

**Review by moseskamira — APPROVED** [review]
LGTM

**Review by Opio-Emmanuel-Omona — APPROVED** [review]
LGTM

**Pull request merged**